streams/src/test/java/org/apache/kafka/streams/KafkaStreamsTest.java

```diff
@@ -289,6 +289,18 @@ public void testCannotCleanupWhileRunning() throws Exception {
         }
     }
 
+    @Test
+    public void testToString() {
+        streams.start();
+        String streamString = streams.toString();
+        streams.close();
+        String appId = streamString.split("\\n")[1].split(":")[1].trim();
+        Assert.assertNotEquals("streamString should not be empty", "", streamString);
+        Assert.assertNotNull("streamString should not be null", streamString);
+        Assert.assertNotEquals("streamString contains non-empty appId", "", appId);
+        Assert.assertNotNull("streamString contains non-null appId", appId);
```

Comment:
Current implementation strips the app-id from streamString to check whether it is non-null and non-empty whereas `streamsString.contains("appId")` will just look for the string "appId" which may or may not be the actual app-id. So IMO the current implementation seems more accurate for assert. What do you think?